scripts/vet-proto.sh

```diff
@@ -32,7 +32,7 @@ elif [[ "$#" -ne 0 ]]; then
 fi
 
 # - Check that generated proto files are up to date.
-go generate google.golang.org/grpc/... && git status --porcelain 2>&1 | fail_on_output || \
+go generate ./... && git status --porcelain 2>&1 | fail_on_output || \
```

Comment:
Again why this change? If the reason is to regenerate protos in submodules, then we should probably be explicit about it.

Reply:
Same for the other `./...` change that Arjan discovered

Reply:
Let's use the other thread to get to the bottom of this.